src/backend/distributed/commands/table.c

```diff
@@ -1268,6 +1269,9 @@ RelationIdListContainsPostgresTable(List *relationIdList)
 static void
 ConvertPostgresLocalTablesToCitusLocalTables(AlterTableStmt *alterTableStatement)
 {
+	/* as we're having a table type conversion, dont need warning messages */
+	InTableTypeConversionFunctionCall = true;
```

Comment:
I still see the unnecessary warnings, where we attaching partitions should not show the warning?

```SQL

SET citus.use_citus_managed_tables TO ON;
CREATE TABLE parent_2 (a INT UNIQUE) PARTITION BY RANGE(a);
CREATE TABLE parent_1 (a INT UNIQUE) PARTITION BY RANGE(a);

SET citus.use_citus_managed_tables TO OFF;
CREATE TABLE parent_2_child_1 (a int);
CREATE TABLE parent_1_child_1 (a int);

CREATE VIEW v1 AS SELECT * FROM parent_2_child_1;
CREATE VIEW v2 AS SELECT * FROM parent_1_child_1;

alter table parent_1 attach partition parent_1_child_1 default ;
WARNING:  "view v2" has dependency to "table parent_1_child_1" that is not in Citus' metadata
DETAIL:  "view v2" will be created only locally
HINT:  Distribute "table parent_1_child_1" first to distribute "view v2"


```

(probably same issue happens with fkey conversions as well)